src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -288,4 +285,44 @@ private function determineSymfonyState()
 
         return $versionState;
     }
+
+    /**
+     * Gets the names, paths and installed versions of the bundles enabled in
+     * the application.
+     *
+     * @return array
+     */
+    private function getBundleData()
+    {
+        $bundles = array();
+
+        try {
+            $composerLockPath = $this->kernel->getRootDir().'/../composer.lock';
+            $composerLock = json_decode(file_get_contents($composerLockPath), true);
+            foreach (array_merge($composerLock['packages'], $composerLock['packages-dev']) as $dependency) {
+                $dependencyNameParts = explode('/', $dependency['name']);
+                $installedDependencies[$dependencyNameParts[1]] = $dependency['version'];
```

Comment:
this assumes that the non-vendor part of package names for bundles is unique, which is totally wrong. See ``sonata-project/core-bundle`` vs ``contao/core-bundle`` for instance.